### DISCUSSION TITLE
[2024-09-19] APAC & EMEA Building DevRel Foundation Community Call

### DISCUSSION THREAD

**Discussion by anajsana** [DevRel-Foundation/governance#66] [Meeting Notes]
> Part of our community calls are being used as Open Office Hours to reach community consensus on the foundation's next phases and address questions and blockers on the DevRel Challenges and Frequently Asked Questions. If you would like to get involved, this is a safe and welcoming space to start!

### 🧹 Housekeeping
- Welcome
  The Developer Relations Foundation mission:

  > The mission of the Developer Relations Foundation project is to elevate the professional practice of developer relations and increase awareness of it as a driver of business value.
- Code of conduct ([link](https://github.com/DevRel-Foundation/governance/blob/main/code_of_conduct.md)). 
Instances of abusive, harassing, or otherwise unacceptable behavior, please report at conduct@dev-rel.org
- Intro to roles: facilitator, moderator, scribe

### 🤔 Open Office Hour topics

Time to listen to the voices of community members, answer questions, and tackle blockers together. Topics can be related to:

1. Review existing proposals filed by the community and open discussions:
  - https://github.com/DevRel-Foundation/governance/discussions/61
  - https://github.com/DevRel-Foundation/governance/discussions/63
2. [Survey Responses Progress](https://docs.google.com/presentation/d/1gI_Pz5J5e8UUKlRGf3vQXWVL6dgAfjsY0PZXjkNvKk8/edit?usp=sharing)
3. Open discussion: please bring your discussion topics!

## Essential Information

* 🕛 Thursday 18, 13:00 - 14:00 pm CEST
* 📝 [g.doc agenda](https://docs.google.com/document/d/1I5dX4RbvJyt4Z2dTWsS1BjmzRQV8RHOJ5klJ_tZa_eg/edit?usp=sharing)

| Frequency | Duration | Zoom link | Join Discord |
| -- | -- | -- | -- |
| bi-weekly | 1 hour | [Here](https://zoom.us/j/97622449945?pwd=K2eKlYuqZwaYkzuH9I5zzrErLYf44M.1) | [Join](https://discord.gg/Vc6Qytc9vE) |

**Comment by leggetter** [comment]
# Meeting notes

🎥  [Recording](https://drive.google.com/file/d/1OfnbWbcZTIFTITJjIWS1XkYqUMmb51ow/view?usp=drive_link)

## Participants (14)

1. Jonathan LeBlanc
2. Phil Leggetter
3. Ana Jiménez
4. Wesley Faulkner
5. Yiga
6. Johannes Dienst
7. Raul Jimenez
8. Ram Iyengar
9. Atsushi
10. Manish Tyagi
11. Tochuk Wun
12. Sparsh Kesari
13. Anita
14. ?

## Workstreams

Jonathan LeBlanc shared that plans of a project-managed approach to workstreams will be shared in near future meetings including timelines, and plans to get working groups in place.

Broad workstreams are:

1. Broaden reach to communities and increase community representation
15. Marketing the DRF Website (eng, design)
16. Project management for phased approach (covered later)
17. Building working groups & getting them started
18. Governance e.g. code of conduct, licensing

## "DevRel Practitioner" Definition - What outlines the criteria? #63

[FinOps definition](https://github.com/finopsfoundation/foundation/blob/main/practitioner_definition.md#finops-practitioner-definition) was shared.

- SK shared a more inclusive definition would be preferred.
- RJ felt there should be a point where there is a clear exclusion.
- JL asked where the line should be drawn
- RJ suggested DevRel should work with Sales but not do Sales
  - Maybe the activities undertaken by a practitioner would be a better way to define what was in scope of DevRel instead of job titles
- Anita asked if DevRel consultants would be excluded?
- JL shared that the feedback from the AMER call was that you should not be able to sell within the DRF
- JL asked where the we should draw the line of selling
- PL shared that some communities (e.g. DevRel Collective) have dedicated channels for selling or promoting
- JL we could create a draft definition and work to get feedback
- WF suggested defining the definition could be a DRF project
- RJ indicated support for WF's proposal
- JL shared that the Definition would fit well with the "Common Language" section of the [challenges](https://github.com/DevRel-Foundation/governance/blob/main/challenges.md)
- WF posed that maybe there's a segmentation of definitions across DevRel Practioner and DevRel Professional

## A Phased Approach for Leading DevRel Community Initiatives #61

- WF: still very much in the discussion phase. Please provide feedback.
  - Outstanding items:
    1. When the plan will be locked in. Should be announced soon.
    2. Lunch phase 1 of plan.

RJ: in the last [EMEA & APAC session](https://github.com/DevRel-Foundation/governance/discussions/62#discussioncomment-10559088), some feedback was shared. What is the best way to ensure that is captured? RJ shared that they would comment on #61 

## Survey responses

AJ walked through the [survey responses slidedeck](https://docs.google.com/presentation/d/1gI_Pz5J5e8UUKlRGf3vQXWVL6dgAfjsY0PZXjkNvKk8/edit?usp=sharing)

Action: AJ to create GitHub Discussion to discuss the responses.

Atsushi suggested translating the survey into Japanese.

JL suggested translating to Portuguese.

## Additional discussions

### DRF community onboarding

RJ asked if there was a defined way to onboard into the DRJ community
JL shared that dev-rel.org will be the entry point to onboarding in future.
